Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -789,6 +789,8 @@ - (BOOL)isFCMAutoInitEnabled {
 
 // Actually makes InstanceID instantiate both the IID and Token-related subsystems.
 - (void)start {
+  FIRInstanceIDLoggerInfo(kFIRInstanceIDSDKVersion, @"FIRInstanceID library version %@",
```

Comment:
Can we make this debug? I don't think we should add too many more logs to startup, it's already quite busy with logs.